src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -103,20 +104,37 @@ private IEnumerable<CodeRefactoringProvider> GetProviders(Document document)
             bool isBlocking,
             CancellationToken cancellationToken)
         {
+            var extensionManager = document.Project.Solution.Workspace.Services.GetService<IExtensionManager>();
+            var isPerProviderLoggingEnabled = RoslynEventSource.Instance.IsEnabled(EventLevel.Informational, EventKeywords.None);
+
             using (Logger.LogBlock(FunctionId.Refactoring_CodeRefactoringService_GetRefactoringsAsync, cancellationToken))
             {
-                var extensionManager = document.Project.Solution.Workspace.Services.GetService<IExtensionManager>();
                 var tasks = new List<Task<CodeRefactoring>>();
 
                 foreach (var provider in GetProviders(document))
                 {
                     tasks.Add(Task.Run(
-                        () => GetRefactoringFromProviderAsync(document, state, provider, extensionManager, isBlocking, cancellationToken), cancellationToken));
+                        () => GetRefactoringsAsync(provider), cancellationToken));
                 }
 
                 var results = await Task.WhenAll(tasks).ConfigureAwait(false);
                 return results.WhereNotNull().ToImmutableArray();
             }
+
+            Task<CodeRefactoring> GetRefactoringsAsync(CodeRefactoringProvider provider)
```

Comment:
❔ Does this method get called for all code fix providers, or only ones that declare support for one or more diagnostics present in the current scope?

Reply:
Ah good point. I have pushed another commit to only log information for fixers that can fix a diagnostic in the span.